src/sentry/models/externalactor.py

```diff
@@ -0,0 +1,62 @@
+import logging
+from enum import Enum
+from typing import Optional
+
+from django.db import models
+
+from sentry.db.models import BoundedPositiveIntegerField, DefaultFieldsModel, FlexibleForeignKey
+
+logger = logging.getLogger(__name__)
+
+
+class ExternalProviders(Enum):
```

Comment:
I've moved this enum to a file in https://github.com/getsentry/sentry/pull/25046